README.md

```diff
@@ -1 +1,88 @@
-# node-modules-object-lab
\ No newline at end of file
+# Using Object Module
+
+## Objectives
+
+1. Write a simple non-npm module with module.exports = {}
```

Comment:
I think we can drop the emphasis on non-npm vs. npm modules -- it ends up just being confusing for the student, I think.